packages/react-aria-components/docs/Toolbar.mdx

```diff
@@ -87,46 +78,104 @@ import {Toolbar, Button, ToggleButton, Separator, Popover, MenuTrigger, Menu, Me
   display: flex;
   flex-wrap: wrap;
   gap: 5px;
-  padding: 15px;
-  border: 1px solid var(--separator-color);
-
-  &[data-orientation=vertical] {
-    flex-direction: column;
-  }
 
   &[data-orientation=horizontal] {
     flex-direction: row;
   }
-}
 
-.react-aria-ToggleButton {
-  width: 32px;
+  .react-aria-Group {
+    display: contents;
+  }
+
+  .react-aria-ToggleButton {
+    width: 32px;
+  }
 }
 
+
 .react-aria-Separator {
-  width: 2px;
-  margin: 0px 10px;
   align-self: stretch;
   background-color: var(--separator-color);
+
+  &[aria-orientation=vertical] {
+    width: 1px;
+    margin: 0px 10px;
+  }
 }
 ```
 
 </details>
 
 ## Features
 
-Toolbar provides keyboard navigation and announcements for groups of interactive elements.
-`Toolbar` helps implement these in an accessible way.
+There is no built-in HTML toolbar element. `Toolbar` helps achieve accessible toolbars with arrow key navigation that can be styled as needed.
 
-* **Flexible** – Support for various RAC or Aria Hooks implemented components.
-* **Accessible** – Follows the [ARIA toolbar pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/) with support for keyboard arrow key navigation and is a single tab stop.
-* **Styleable** – Can be styled in any custom way, including extra DOM structure between the Toolbar and the interactive children.
```

Comment:
This one didn't seem super necessary IMO since it is true of all React Aria components.